### PR TITLE
Java Doc Repairs.

### PR DESCRIPTION
**Pull request by RdrOfTheSt0rm**
Minor repairs to warnings givin by javadoc, mostly @see @param.
I intend to continue this work later.

### CODE REVIEW

File: src/freenet/client/async/BaseManifestPutter.java [hunk 1]
```diff
@@ -51,10 +51,12 @@
  * <DT>container mode: <DD>the metadata are inside the root container (the final URI points to an archive)
  * <DT>freeform mode: <DD>the metadata are inserted separately.(the final URI points to a SimpleManifest)
  * </DL>
- * @see {@link PlainManifestPutter} and {@link DefaultManifestPutter}</P>
  * 
- * WARNING: Changing non-transient members on classes that are Serializable can result in 
+ *  WARNING: Changing non-transient members on classes that are Serializable can result in
```
Comment (Thynix) [inline src/freenet/client/async/BaseManifestPutter.java]: This added space shouldn't be here.

File: src/freenet/support/io/NativeThread.java [hunk 2]
```diff
@@ -10,7 +10,8 @@
 /**
  * Do *NOT* forget to call super.run() if you extend it!
  * 
- * @see http://archives.freenetproject.org/thread/20080214.235159.6deed539.en.html
+ * @see "Old link - http://archives.freenetproject.org/thread/20080214.235159.6deed539.en.html"
```
Comment (Thynix) [inline src/freenet/support/io/NativeThread.java]: Why keep the old link if it no longer works?

File: src/freenet/client/async/BaseManifestPutter.java [hunk 3]
```diff
@@ -51,10 +51,12 @@
  * <DT>container mode: <DD>the metadata are inside the root container (the final URI points to an archive)
  * <DT>freeform mode: <DD>the metadata are inserted separately.(the final URI points to a SimpleManifest)
  * </DL>
- * @see {@link PlainManifestPutter} and {@link DefaultManifestPutter}</P>
  * 
- * WARNING: Changing non-transient members on classes that are Serializable can result in 
+ *  WARNING: Changing non-transient members on classes that are Serializable can result in 
  * restarting downloads or losing uploads.
+ * </P>
+ * @see freenet.client.async.PlainManifestPutter PlainManifestPutter, freenet.client.async.DefaultManifestPutter DefaultanifestPutter
```
Comment (Thynix) [inline src/freenet/client/async/BaseManifestPutter.java]: DefaultanifestPutter should be DefaultManifestPutter.

**Comment by Thynix** [comment]
This commit message doesn't meet the [commit message standards](https://wiki.freenetproject.org/Coding_style#Commit_Messages). In particular, it is not phrased as a command to the codebase, and it ends with a period. How about something more like `JavaDocs: fix broken links`?

**Comment by Thynix** [comment]
Thanks for doing this! The JavaDocs have certainly been falling into disrepair.

File: src/freenet/support/io/NativeThread.java (hunk 2 above)
Comment (RdrOfTheSt0rm) [inline src/freenet/support/io/NativeThread.java]: I left it there as i wasnt 100% that i have listed the correct new link.
You want it removed?

**Pull request closed without merging**